protos/local/common.proto

```diff
@@ -38,10 +38,11 @@ message NamespacedWorkflowExecution {
 // Defines how an activity or workflow should be retried in the event of failure, timeout, etc.
 message RetryPolicy {
     // Interval of the first retry. If backoff_coefficient is 1.0 then it is used for all
-    // retries.
+    // retries. If this is not specified, there will be *no retrying*. Lang SDKs should require
```

Comment:
Because the right way to say that is max attempts 1.

Lang sdks should just default this to some reasonable value always, rather than allowing the odd situation of max attempts being > 1 but this being unset.

Reply:
Looks like that's what java does: https://github.com/temporalio/sdk-java/blob/820627bc6050dfba13c252f46c3518715004a02e/temporal-sdk/src/main/java/io/temporal/common/RetryOptions.java#L112